### PR TITLE
Switched to scala 2.11.6 / SBT 0.13.8

### PR DESCRIPTION
**Pull request by alexarchambault**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,20 +7,23 @@
...
+ENV SCALA_VERSION 2.11.6
+ENV SBT_VERSION 0.13.8
+
 # Install Scala
 RUN \
   cd /root && \
-  curl -o scala-2.11.4.tgz http://downloads.typesafe.com/scala/2.11.4/scala-2.11.4.tgz && \
-  tar -xf scala-2.11.4.tgz && \
-  rm scala-2.11.4.tgz && \
+  curl -o scala-$SCALA_VERSION.tgz http://downloads.typesafe.com/scala/2.11.6/scala-$SCALA_VERSION.tgz && \
```
Comment (hseeberger) [inline Dockerfile]: You forgot one `s/2.11.6/$SCLAA_VERSION` ;-)

Comment (alexarchambault) [inline Dockerfile]: oops :-)

**Comment by hseeberger** [comment]
Nice! Thanks a lot!

**Comment by hseeberger** [comment]
Hmm ... did you test that?

On my Mac I get 

```
Step 3 : RUN cd /root &&   curl -o scala-$SCALA_VERSION.tgz http://downloads.typesafe.com/scala/$SCALA-VERSION/scala-$SCALA_VERSION.tgz &&   tar -xf scala-$SCALA_VERSION.tgz &&   rm scala-$SCALA_VERSION.tgz &&   echo >> /root/.bashrc &&   echo 'export PATH=~/scala-$SCALA_VERSION/bin:$PATH' >> /root/.bashrc
 ---> Running in 57fccfdbca0a
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   231    0   231    0     0    552      0 --:--:-- --:--:-- --:--:--   553
tar: This does not look like a tar archive

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
INFO[0001] The command [/bin/sh -c cd /root &&   curl -o scala-$SCALA_VERSION.tgz http://downloads.typesafe.com/scala/$SCALA-VERSION/scala-$SCALA_VERSION.tgz &&   tar -xf scala-$SCALA_VERSION.tgz &&   rm scala-$SCALA_VERSION.tgz &&   echo >> /root/.bashrc &&   echo 'export PATH=~/scala-$SCALA_VERSION/bin:$PATH' >> /root/.bashrc] returned a non-zero code: 2
```

**Comment by alexarchambault** [comment]
Yes I did... Let me check again...

**Comment by alexarchambault** [comment]
Ok, I hadn't checked the very last fix. The typo is in it.

**Pull request merged**